lib/index.js

```diff
@@ -67,5 +69,33 @@ function generateKarmaConfig() {
     karmaConfig.preprocessors[testSuite] = DEFAULT_PREPROCESSORS;
   });
 
+  if(!('httpModule' in karmaConfig)) {
+    // override httpModule to handle cluster issues
+    karmaConfig.httpModule = httpModule;
+  }
+
+  if(coverage || coverageCi) {
+    karmaConfig.coverageIstanbulReporter = {
+      reports: ['text', 'text-summary', 'lcovonly'],
+      fixWebpackSourcePaths: true,
+    };
```

Comment:
This makes it harder to configure another way ... but I'm ok with it -- we have found that we don't change this config anyway.

Reply:
Yeah, I know.  Trading off the common path for some flexibility.  Unsure what we would practically change on a per-package basis so figured this would be ok until we find out what sort of customization might be needed.